etc/systemd/bloop.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Bloop Scala build server
+
+[Service]
```

Comment:
Can we redirect logs to a file? Maybe `/var/log` if we can or a log file called `server.log` in the bloop installation target?

Reply:
I think the standard way to do this in `systemd` is to redirect `stdout`/`stderr` to system log and let it be handled by `journalctl`. The issue with `/var/log` is that the user running Bloop most likely won't have permissions to write there.

Reply:
Please see https://github.com/scalacenter/bloop/pull/440/commits/e2a4ff525da6a92e95598aa04d3f5eeb98592ee6